.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```

Comment:
I am not a fan of this, since this moves us to the full-VM environment from the container service: https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments.   Do we need sudo to install git-lfs?  Can it be done locally?